tests/bootstrap.php

```diff
@@ -1,54 +1,37 @@
 <?php
+/**
+ * This file handles initializing MediaWiki stack and setting global flags
+ * before handing over execution to PHPUnit that executes the tests
+ * provided in its configuration file set via the command line.
+ */
+
+error_reporting( E_ALL );
+ini_set( 'display_errors', 1 );
+
 $wgRunningUnitTests = true; // don't include DevBoxSettings when running unit tests
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)